src/routes/auth/guards/auth.guard.ts

```diff
@@ -36,7 +46,7 @@ export class AuthGuard implements CanActivate {
     }
 
     try {
-      this.authRepository.verifyToken(accessToken);
+      request.auth = this.authRepository.verifyToken(accessToken);
```

Comment:
`auth` seems a bit generic. I think `accessToken` is more descriptive and more representative of what the property holds.

Reply:
Renamed in f047089a1153d5234508a9c1b4d45e85255cd483.